frontend/scripts/clean_orval_generated.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
IMHO we should start using auto-generated hooks and deprecate all manually created files in `frontend/hooks/api`. There are things that can go wrong in the beginning, like forgetting to refetch, but it's much more safe then what we currently have. I think automating this is a step backwards. @FredrikOseberg

Reply:
Currently the generate script alone requires 2 additional manual steps for the code to be usable.  I fail to see how automating them is a step backwards